app/src/main/java/com/example/pocketcloset/Fragments/WardrobeFragment.java

```diff
@@ -1,65 +1,224 @@
+           @Override
+           public void onButtonChecked(MaterialButtonToggleGroup group, int checkedId, boolean isChecked) {
+               if (checkedId == R.id.btnTop) {
+                   adapter.clear();
+                    queryClothes("Top");
+                   adapter.notifyDataSetChanged();
+               }
+               if (checkedId == R.id.btnBottoms) {
+                   adapter.clear();
+                   queryClothes("Bottom");
+                   adapter.notifyDataSetChanged();
+
+               }
+
+               if (checkedId == R.id.btnAccessories) {
+                   adapter.clear();
+                   queryClothes("Earrings");
+                   adapter.notifyDataSetChanged();
+
+               }
+               if (checkedId == R.id.btnHeadwear) {
+                   adapter.clear();
+                   queryClothes("Headwear");
+                   adapter.notifyDataSetChanged();
+
+               }
+               if (checkedId == R.id.btnShoes) {
+                   adapter.clear();
+                   queryClothes("Shoes");
+                   adapter.notifyDataSetChanged();
+
+               }
+               if (checkedId == R.id.btnOverwear) {
+                   adapter.clear();
+                   queryClothes("Overwear");
+                   adapter.notifyDataSetChanged();
+
+               }
+           }
+       });
+
+
+
+        queryClothes();
+
+
+
+
+
+
+
+    }
+
+    protected void queryClothes() {
+        // specify what type of data we want to query - Post.class
+        ParseQuery<Clothing> query = ParseQuery.getQuery(Clothing.class);
+        // include data referred by user key
+        //query.include(Clothing.CLOTHING_TYPE);
+        query.include(Clothing.KEY_CLOTHING_IMAGE);
+        query.whereEqualTo(Clothing.KEY_USER, user);
+        // limit query to latest 20 items
+        query.setLimit(20);
+        // order posts by creation date (newest first)
+        query.addDescendingOrder("createdAt");
+        // start an asynchronous call for posts
+        query.findInBackground(new FindCallback<Clothing>() {
+            @Override
+            public void done(List<Clothing> clothes, ParseException e) {
+                // check for errors
+                if (e != null) {
+                    Log.e("WardrobeFragment", "Issue with getting clothing", e);
+                    return;
+                }
+
+                // for debugging purposes let's print every post description to logcat
+                for (Clothing clothing : clothes) {
+                //    Log.i("WardrobeFragment", "Clothing: " + clothing.getClothingType() );
+                }
+
+                // save received posts to list and notify adapter of new data
+               // adapter.clear();
+                allClothing.addAll(clothes);
+          //      Log.i("WardrobeFragment", "Clothing: " + allClothing.size());
+                adapter.notifyDataSetChanged();
+            }
+        });
```

Comment:
This query api looks like it's meant to be fluent. You can likely chain the calls to make it more like english/sql
```
ParseQuery.getQuery(Clothing.class)
  .include(Clothing.KEY_CLOTHING_IMAGE)
  .whereEqualTo(Clothing.KEY_USER, user)
  .setLimit(20)
  .addDescendingOrder("createdAt")
  .findInBackground(new FindCallback<Clothing>() { ... });
```